subhub/api/payments.py

```diff
@@ -98,7 +97,7 @@ def check_stripe_subscriptions(customer: str) -> list:
         return []
 
 
-def sources_to_remove(subscriptions: list, customer: str) -> None:
+def sources_to_remove(subscriptions: list, customer: stripe.Customer) -> None:
```

Comment:
consider`from stripe import Customer` to reduce all of the `stripe.Customer` -> `Customer`